Flaky requests

Pretty sure I can fix this after doing a bit of reading but I want it documented that this happens on a lot of systems. 
"""
----------------------------------------------------------------------------------------------------------- Captured stdout call ------------------------------------------------------------------------------------------------------------
 * Serving Flask app 'financify_api.__app__'
 * Debug mode: off
========================================================================================================== short test summary info ==========================================================================================================
FAILED tests/test_statements.py::test_asset_resource_errors[users_schema] - requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /assets (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5c21de9d20>: Failed to esta...
FAILED tests/test_users.py::test_user_success[users_schema] - requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /users (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5c21725a80>: Failed to estab...
FAILED tests/test_users.py::test_user_errors[users_schema] - requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /users (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5c216c0040>: Failed to estab...
FAILED tests/test_users.py::test_user_restriction[users_schema] - requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /users (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5c22c5aad0>: Failed to estab...
======================================================================================================== 4 failed, 2 passed in 3.94s ==================
"""

--- a/README.md
+++ b/README.md
@@ -21,7 +21,7 @@ I started this project as a way to learn the basics of REST APIs and fullstack c
 3. Run setup entry point to configure .env and .db
     - `poetry run setup`
 4. Run unit tests to make sure everything runs correctly
-    - `poetry run pytest
+    - `poetry run pytest`
     - Note: I am still working out a `Max retries exceeded with url: ` error that shows up sometimes. If that happens move on to the next section.
 
 ## Running the flask server

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -20,6 +20,7 @@ def admin_access_app(request: pytest.FixtureRequest) -> Iterator[Union[Process, 
     _ = create_api(app)
     app_process = Process(target=app.run, kwargs={"debug": False})
     app_process.start()
+    time.sleep(0.5)
     yield app_process
     app_process.terminate()
     os.remove(test_db)
@@ -35,6 +36,7 @@ def base_access_app(request: pytest.FixtureRequest) -> Iterator[Union[Process, s
     _ = create_api(app)
     app_process = Process(target=app.run, kwargs={"debug": False})
     app_process.start()
+    time.sleep(0.5)
     yield app_process
     app_process.terminate()
     os.remove(test_db)

